kbase-extension/static/kbase/js/widgets/appWidgets2/inputUtils.js

```diff
@@ -3,37 +3,35 @@ define(['common/html', 'common/events', 'common/ui'], (html, Events, UI) => {
 
     const t = html.tag,
         div = t('div'),
-        span = t('span'),
+        b = t('b'),
         button = t('button'),
         ui = UI.make({ node: document.body });
 
-    function showMessageDialog(id) {
+    function showMessageDialog(title, id) {
         ui.showInfoDialog({
-            title: 'MESSAGE TITLE',
+            title,
             body: 'Message id: ' + id,
         });
     }
 
     function buildMessageAlert(messageDef) {
-        const events = Events.make({ node: document.body }),
+        const events = Events.make(),
```

Comment:
After browsing through the codebase, I've found this only gets called when an input widget is configured to use it. Which, as far as I can tell, is never. Also, it's effectively copied into the 4 different `fieldWidget` variants where it actually gets run. As much as that irks me, fixing that is out of scope of this PR. I'll make a ticket for another time. Sigh.

Reply:
😖